index.js

```diff
@@ -16,7 +16,7 @@
  */
 'use strict';
 
-const defaultUrl = 'https://voice-memos.appspot.com';
+const defaultUrl = 'https://m.flipkart.com';
```

Comment:
do we want `m.flipkart.com` or allow it to redirect from `flipkart.com` if it wants to?